CMakeLists.txt

```diff
@@ -263,6 +264,14 @@ if(TRITON_BUILD_PYTHON_MODULE AND NOT WIN32)
   target_link_libraries(triton ${CUTLASS_LIBRARIES} ${PYTHON_LDFLAGS})
 endif()
 
+if (NOT "X${TRITON_CODEGEN_BACKENDS}" STREQUAL "X")
```

Comment:
is it to check it `TRITON_CODEGEN_BACKENDS` is empty? Surely there should be a better way...

Reply:
Yes. I refined the code by checking the length of `TRITON_CODEGEN_BACKENDS`.